lib/monitoring/aws-elasticache/ElastiCacheClusterMonitoring.ts

```diff
@@ -157,6 +172,20 @@ export class ElastiCacheClusterMonitoring extends Monitoring {
       this.cpuUsageAnnotations.push(createdAlarm.annotation);
       this.addAlarm(createdAlarm);
     }
+
+    for (const disambiguator in props.addRedisEngineCpuUsageAlarm) {
```

Comment:
Should throw an error if this is attempted when it's not a Redis cluster.

Reply:
done (see above)